streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImpl.java

```diff
@@ -230,38 +288,23 @@ public Sensor addLatencyAndThroughputSensor(final String scopeName,
                                                 final Sensor.RecordingLevel recordingLevel,
                                                 final String... tags) {
 
-        return addLatencyAndThroughputSensor(null,
-                                             scopeName,
-                                             entityName,
-                                             operationName,
-                                             recordingLevel,
-                                             tags);
-
-    }
-
-    public Sensor addLatencyAndThroughputSensor(final String taskName,
-                                                final String scopeName,
-                                                final String entityName,
-                                                final String operationName,
-                                                final Sensor.RecordingLevel recordingLevel,
-                                                final String... tags) {
-
```

Comment:
The task-scoped `addLatencyAndThroughputSensor` is unused now, so I've removed it.

FWIW, the non-task-scoped `addLatencyAndThroughputSensor` is unused in our code base, but it's part of the public `StreamsMetrics` interface, so I've left it and its tests. Perhaps we can have a KIP some time to remove it.

Reply:
`addLatencyAndThroughputSensor` and others in `StreamsMetrics` are for users to register their customized metrics, so we cannot remove it..

Reply:
Note that for now these customized metrics will only be add on the "thread-level" though the tag map contains two: the scopeName = entityName, and threadName = [thread-id].

Reply:
I didn't follow that last comment. These public sensor methods create both a thread-level "operationName" sensor and a "entityName"-level "operationName" sensor.